### ISSUE TITLE
ops.unary_union segfaults on empty Polygons

### ISSUE THREAD

**Issue by Lyle-Alloy** [shapely/shapely#1333]
## Expected behavior and actual behavior.

Expected that shapely doesn't segfault.

## Steps to reproduce the problem.

Running:
```python
>>> from shapely import geometry, ops
>>> a = geometry.Polygon()
>>> a.is_valid
True
>>> ops.unary_union(a)

```
results in:
```
fish: 'python' terminated by signal SIGSEGV (Address boundary error)
````

## Operating system
MacOS Catalina 10.15.7

## Shapely version and provenance

main (f048c1816ad5782c2e70b42c55e5d74fd19ce7e7) installed via git pull + python setup.py develop

**Comment by jorisvandenbossche** [comment]
Which version of GEOS are you using? This should be fixed with GEOS 3.10.2 I think

**Comment by jorisvandenbossche** [comment]
Going to close this as fixed in GEOS (and our packages now include a more recent version of GEOS)
